packages/httpsnippet-client-api/test/__datasets__/full-many-query-params/har.json

```diff
@@ -32,11 +32,11 @@
           ],
           "cookies": [
             {
-              "name": "foo",
+              "name": "foo-cookie",
```

Comment:
Changed these cookie names off `foo` and `bar` because this, and the `full`, test have those already as parameters and `api` doesn't support differing parameter types that share the same name.